apps/backend/src/services/polling_service.py

```
@@ -25,13 +25,17 @@
 logger = logging.getLogger(__name__)
 
 class PollingService:
-    def __init__(self, db_session: AsyncSession):
-        self.db = db_session
+    def __init__(self):
         self.ssh_client = get_ssh_client()
         self.settings = get_settings()
         self.polling_tasks: Dict[UUID, asyncio.Task] = {}
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)